DependencyInjection/Compiler/ResourcePass.php

```diff
@@ -62,6 +58,23 @@ public function process(ContainerBuilder $container)
     }
 
     /**
+     * Adds a list of operations.
+     *
+     * @param ContainerBuilder $container
+     * @param string           $serviceId
+     * @param array            $methods
+     * @param bool             $collection
+     *
+     * @return Reference[]
+     */
+    private function createOperations(ContainerBuilder $container, $serviceId, $methods, $collection)
+    {
+        return array_map(function ($method) use ($container, $serviceId, $collection) {
```

Comment:
Question if habit, avoid temporary variable and gain 3 lines of code.

I can revert it if you prefer something more explicit